src/DynamoCoreWpf/ViewModels/Core/ShortcutToolbarViewModel.cs

```diff
@@ -15,14 +15,22 @@ internal class ShortcutToolbarViewModel : ViewModelBase
         private AuthenticationManager authManager;
 
         private int notificationsNumber;
+        private bool viewButtonsText;
 
         public ShortcutToolbarViewModel(DynamoViewModel dynamoViewModel)
         {
             NotificationsNumber = 0;
             authManager = dynamoViewModel.Model.AuthenticationManager;
             ValidateWorkSpaceBeforeToExportAsImageCommand = new DelegateCommand(dynamoViewModel.ValidateWorkSpaceBeforeToExportAsImage);
             SignOutCommand = new DelegateCommand(authManager.ToggleLoginState);
-            authManager.LoginStateChanged += (o) => { RaisePropertyChanged(nameof(LoginState)); };
+            authManager.LoginStateChanged += (o) => { RaisePropertyChanged(nameof(LoginState)); };            
+            dynamoViewModel.WindowRezised += DynamoViewModel_WindowRezised;
+            ViewButtonsText = true;
+        }
+
+        private void DynamoViewModel_WindowRezised(object sender, System.EventArgs e)
+        {
+            ViewButtonsText = !(bool)sender;
```

Comment:
It would be good to make sure only a bool value is passed here, otherwise the casting will throw an exception.

Reply:
We can use `Show/HideText`, it is not guaranteed a button either, right? One of the case is a menu item